docs/contributor/04-30-ci-cd.md

```diff
@@ -1,51 +1,71 @@
 # CI/CD
 The Istio module contains several pipelines that run on pull requests, on the main branch, on the release branch, and periodically. They perform various tasks such as running a linter, executing unit tests with code coverage, generating artifacts for development and production purposes, and running integration tests. The integration tests ensure the functional correctness of Kyma Istio Operator and verify if upgrading to the new version of Kyma Istio Operator is possible.
 
-## Pipelines running on pull requests
+## Pipelines running on development pull requests
```

Comment:
We should ahve it similar to [api-gateway](https://github.com/kyma-project/api-gateway/blob/main/docs/contributor/04-30-ci-cd.md#pipelines-running-on-pull-requests-to-the-main-branch)

```suggestion
## Pipelines running on pull requests to the `main` branch
```